### DISCUSSION TITLE
Excluding testfiles?

### DISCUSSION THREAD

**Discussion by parkerhancock** [github-linguist/linguist#6024] [Classification] [answered]
I am the developer of `patent_client`, a python library for analyzing patent data. The test suite is large, and has a lot of HTML fixtures that the library parses through. Because the volume of HTML is so much larger than the actual python code, my library is being misclassified as an HTML project. What do I need to do to fix it?

![Patent Client Logo](https://github.com/parkerhancock/patent_client/raw/master/docs/_static/patent_client_logo.svg)

[PyPI](https://pypi.org/project/patent-client/) | [GitHub](https://github.com/parkerhancock/patent_client) | [Docs](https://patent-client.readthedocs.io/en/latest/)

Thanks!

Parker

**Comment by lildude** [comment, marked as answer]
> What do I need to do to fix it?

Implement an [override](https://github.com/github/linguist/blob/master/docs/overrides.md) 😉
